Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Dockerfile for setting up reviewdog and ktlint
+
+FROM ubuntu:latest
+
+RUN apt-get update && apt-get install -y curl
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Delete the apt-get lists after installing something](https://app.codacy.com/manual/davidmerrick/AdventOfCode2019/pullRequest?prid=4961687)